scripts/irods/test/test_all_rules.py

```diff
@@ -489,6 +486,46 @@ def test_writeLine_config_first_3477(self):
             IrodsController().start()
 
 
+    @unittest.skipUnless(plugin_name == 'irods_rule_engine_plugin-irods_rule_language', 'only applicable for irods_rule_language REP')
+    def test_msiServerMonPerf_default_3736(self):
+        path_to_irodsServerMonPerf = '/'.join([paths.home_directory(), 'msiExecCmd_bin', 'irodsServerMonPerf'])
```

Comment:
Consider using [`os.path.join`](https://docs.python.org/3.5/library/os.path.html#os.path.join) instead.

Reply:
Oh, good point. I'd gotten into this habit due to PRC running on Windows and then trying to construct iRODS logical paths. This is referring to a path on the local filesystem, so `os.path.join` should be better here.

Reply:
Done